pkgs/flox/default.nix

```diff
@@ -143,8 +143,10 @@ in
           gnused
         ];
 
-      # test all our crates (include the libraries)
-      cargoTestExtraArgs = "--workspace";
+      # Tests are disabled inside of the build because the sandbox prevents
+      # internet access and there are tests that require internet access to
+      # resolve flake references among other things.
+      doCheck = false;
```

Comment:
I'd like to do tests still. Just do unit tests (ie: don't require internet). And do other kinds of tests that require connectivity in another way.

Reply:
We are doing tests, they're just not done inside `flox build`

Reply:
This got resolved by doing the tests that need to be outside in a dev shell, and the others inside the build.

Reply:
That's not actually what we're doing, we're doing all of the tests outside of the sandbox right now unless something else has changed.